app/src/main/java/com/example/messengerAndroid/ui/myContacts/contactsViewModel/ContactsViewModel.kt

```diff
@@ -31,9 +31,17 @@ class ContactsViewModel : ViewModel() {
 
     fun addNewUser(name: String, job: String) {
         _contactsLiveData.value = _contactsLiveData.value?.toMutableList()?.apply {
-            add(User(_contactsLiveData.value!!.size.toLong(), "...", name, job))
+            add(User(_contactsLiveData.value!!.size.toLong(), "", name, job))
         }
     }
 
+    fun getUserPosition(user: User): Int {
+        return contactsLiveData.value?.indexOf(user) ?: -1
+    }
+
+    fun getUser(position: Int): User? {
+        return contactsLiveData.value?.toMutableList()?.get(position)
```

Comment:
I believe that if the position is less than zero then you might have an exception here. It`s better to handle it